src/mongocxx/client.cpp

```diff
@@ -55,6 +55,16 @@ void* client::implementation() const {
     return _impl->client_t;
 }
 
+void client::read_concern(class read_concern rc) {
+    libmongoc::client_set_read_concern(_impl->client_t, rc._impl->read_concern_t);
+}
+
+class read_concern client::read_concern() const {
+    auto rc = libmongoc::client_get_read_concern(_impl->client_t);
+    return (class read_concern){
```

Comment:
Do we want to cache the read concern to avoid this copying dance?  I'm leaning towards what you have here, so we can avoid hanging more things on the client object, but just thought I'd raise the question.